web-client/src/presenter/computeds/formattedTrialSessionDetails.test.ts

```diff
@@ -377,6 +378,25 @@ describe('formattedTrialSessionDetails', () => {
       });
     });
 
+    it('should be false when trial session start date is in the future, it is NOT closed, the user is a judge role', () => {
```

Comment:
I wonder if it is worth renaming to "should not allow judge to edit a trial session". Otherwise, a dev looking at this might think, "But maybe a judge should be able to edit a trial session if it _is_ closed" etc.

Reply:
Good point, implementing!